clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress

```diff
@@ -199,6 +213,9 @@ re_reg_cmd = [SIPP_BIN,
 if args.messages:
     caller_file = "message_sender.xml"
     callee_file = "error_message_receiver.xml" if args.with_errors else "message_receiver.xml"
+elif args.wildcards:
+    caller_file = "caller.xml"
```

Comment:
Do we actually need `wildcard_caller.xml`? Is it not the same, but with a P-Profile-Key? Can't we just have [field4] as `P-Profile-Key: <blah>\n` or whatever and use the same file?